src/docs/patterns/lists.mdx

```diff
@@ -1,56 +1,193 @@
-<DemoCard repository={'status-list'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Fstatus-list%2Fstatus-list.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/status-list/index.tsx&initialpath=status-list'} ionic reactNative/>
+<ImageGrid
+    images={[SimpleListImage]}
+    regularWidth
+    caption={[
+        '1: Lists can be as simple as data-value pairs.',
+        '2: You may add avatars to distinguish each list elements.',
+    ]}
+/>
+<DemoCard
+    repository={'data-list'}
+    angular={
+        'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Fdata-list%2Fdata-list.component.ts'
+    }
+    react={
+        'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/data-list/index.tsx&initialpath=data-list'
+    }
+    ionic
+    reactNative
+/>
 
 The simplest type of list simply presents a single line (or multiple lines) of text for each element in the collection.
 
-Commonly in PX Blue applications, list items also show some visual indication of the item's current status (such as online, offline, or an error condition). This is achieved by using status/type icons (avatars) and sometimes a colored left (inset) border.
+## Lists with Status
 
-## Lists with Actions
+<ImageGrid images={[StatusListImage]} regularWidth caption={`Use an icon avatar and / or a status stripe to catch people's attention.`}/>
+<DemoCard repository={'status-list'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Fstatus-list%2Fstatus-list.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/status-list/index.tsx&initialpath=status-list'} ionic reactNative/>
 
-<ImageGrid images={[Image3, Image4, Image5]} />
-<DemoCard repository={'action-list'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Flist%2Faction-list%2Faction-list.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/lists/action-list/index.tsx&initialpath=action-list'} ionic reactNative />
+Commonly in PX Blue applications, list items also show some visual indication of the item's current status, such as online, offline, or an error condition. This is achieved by using avatars and status stripes.
 
-If there are actions that you would like to perform on individual list items, these can be included via icons or a 3-dot action menu (multiple actions) on each list item. Depending on the space available, you typically will want to use the action menu if you have more than three actions that can be taken. Global actions that affect the whole list (such as Add or Clear) should be placed in the list header.
+<ImageGrid
+    images={[IconAlignImage]}
```

Comment:
> maybe a space measurement indicator to show the spacing from the text? 

You mean the kind of indicator we see in the specs section?

Reply:
yeah, just a thought. Moving the dotted line over is probably enough.

Reply:
Moved the dotted line. I like the way it looks right now — I think all the specs should be put together in one section, rather than scattered everywhere.